scripts/popup.js

```diff
@@ -6,21 +6,32 @@ import storage from './storage.js';
       document.getElementById('btn-openair').style.display = 'block';
     }
 
+    if (tab.url.includes('azure')) {
```

Comment:
can you tab.url.includes('azure') || tab.url.includes('jira') and use just one if?